build.sbt

```diff
@@ -1074,7 +1063,7 @@ lazy val examples = http4sProject("examples")
     ),
     // todo enable when twirl supports dotty TwirlKeys.templateImports := Nil,
   )
-  .dependsOn(server.jvm, dropwizardMetrics, theDsl.jvm, circe.jvm, scalaXml /*, twirl*/ )
+  .dependsOn(server.jvm, dropwizardMetrics, theDsl.jvm, circe.jvm /*, twirl*/ )
```

Comment:
Can we remove commented 'twirl' here too?

Reply:
That's done in https://github.com/http4s/http4s/pull/6357/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R1056